src/braket/experimental/autoqasm/operators/__init__.py

```diff
@@ -21,6 +21,7 @@
 # We need to either implement these, or determine they are not needed and remove them.
 
 # Operators below are imported directly from core autograph implementation
+from braket.experimental.autoqasm.autograph.impl.api_core import autograph_artifact  # noqa: F401
```

Comment:
What a weird bug

Reply:
Yeah, this was just one of these weird autograph implementation details that I missed. At least the fix is easy.